oauth.js

```diff
@@ -0,0 +1,12 @@
+// This is where we would put environment ID's, just in case we wanted to sign
+// up with google or something.
+
+// var ids = {
+//   facebook: {
+//     clientID: process.env.FB_clientID,
+//     clientSecret: process.env.FB_clientSecret,
+//     callbackURL: process.env.FB_callbackURL
+//   }
+// };
+
+// module.exports = ids;
```

Comment:
You can add environment variables in Heroku; I've already done it for the DB. Yeah, I'd thought of putting all environment variables somewhere, but if all we have is a db connection it doesn't seem necessary. Maybe you're right and there's no point pushing this file though? I just wanted to show the format for doing authentication if we do end up doing it (if they have gmails or something and want to use that to login, that seems like a reasonable feature to add)